src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests.cs

```diff
@@ -12174,5 +12174,50 @@ public static void Main(string[] args)
                 );
             var comp = CompileAndVerify(compilation, expectedOutput: "whatever");
         }
+
+        [Fact, WorkItem(12996, "https://github.com/dotnet/roslyn/issues/12996")]
+        public void TypeOfAVarPatternVariable()
```

Comment:
I am referring to the tests in OutVarTests.cs

---
In reply to: [75774455](https://github.com/dotnet/roslyn/pull/13274#discussion_r75774455) [](ancestors = 75774455)